src/applications/mhv/secure-messaging/actions/folders.js

```diff
@@ -31,22 +32,25 @@ export const getFolders = () => async dispatch => {
     }
     if (response.errors) {
       const err = response.errors[0];
-      handleErrors(err);
+      dispatch(handleErrors(err));
     }
   } catch (error) {
     const err = error.errors[0];
-    handleErrors(err);
+    dispatch(handleErrors(err));
+    dispatch({
+      type: Actions.Folder.GET_LIST_ERROR,
+    });
   }
 };
 
 export const retrieveFolder = folderId => async dispatch => {
-  const handleErrors = error => {
-    dispatch({
-      type: Actions.Folder.GET,
-      response: null,
-    });
-    dispatch(addAlert(Constants.ALERT_TYPE_ERROR, '', error?.detail));
-  };
+  // const handleErrors = error => {
+  //   dispatch({
+  //     type: Actions.Folder.GET,
+  //     response: null,
+  //   });
+  //   dispatch(addAlert(Constants.ALERT_TYPE_ERROR, '', error?.detail));
+  // };

```

Comment:
Are you planning to keep this?

Reply:
jeez, rushing too fast here, gotta get that out of there